cockpit-frontend/src/components/ScrumMasterForm/ScrumMasterForm.jsx

```diff
@@ -0,0 +1,277 @@
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import FormLabel from '@material-ui/core/FormLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import InputLabel from '@material-ui/core/InputLabel';
+import FormControl from '@material-ui/core/FormControl';
+import Button from '@material-ui/core/Button';
+import CancelIcon from '@material-ui/icons/Cancel';
+import IconButton from '@material-ui/core/IconButton';
+import { showSMFormState, close } from '../Header/HeaderSlice';
+import {
+  SMFormState,
+  setMvp,
+  setJiraId,
+  setEntity,
+  setCycle,
+  setFormIsValid,
+  setImageUrl,
+  imageUrlState,
+  mvpState,
+  jiraIdState,
+  entityState,
+  cycleState,
+} from './ScrumMasterFormSlice';
+import '../../App.scss';
+import useStyles from './styles';
+import MvpService from '../../services/service';
+import { fetchAllMvps } from '../../redux/ormSlice';
+
+export default function ScrumMasterForm() {
+  const classes = useStyles();
+
+  let url = '';
+  let btnClass = [classes.bubbleBox, 'bubble'];
+  btnClass = btnClass.join(' ');
+
+  const [isOpened, setIsOpened] = useState(false);
+  const imageUrl = useSelector(imageUrlState);
+  const mvpName = useSelector(mvpState);
+  const jira = useSelector(jiraIdState);
+  const selectedEntity = useSelector(entityState);
+  const cycle = useSelector(cycleState);
+  const isFormValid = useSelector(SMFormState);
+  const open = useSelector(showSMFormState);
+  const dispatch = useDispatch();
+
+  function openDialog() {
+    setIsOpened(true);
+  }
+  function closeDialog() {
+    setIsOpened(false);
+  }
+  function confirmImageUrl() {
+    if (url !== '') {
+      dispatch(setImageUrl(url));
+      dispatch(setFormIsValid());
+      url = '';
+      closeDialog();
+    }
+  }
+  function removeImageUrl() {
+    dispatch(setImageUrl(null));
+  }
+  function handleChange(event) {
+    url = event.target.value;
+  }
+  async function submit(e) {
+    e.preventDefault();
+    const mvp = {
+      id: jira,
+      name: mvpName,
+      pitch: 'Hello',
+      status: 'inprogress',
+      iterationNumber: cycle,
+      entity: selectedEntity,
+      currentSprint: 2,
+      nbSprint: 0,
+      mvpEndDate: '',
+      mvpStartDate: '',
+      mvpAvatarUrl: imageUrl,
+      jiraProjectId: 10009,
+      jiraBoardId: 1004,
+      location: '',
+      nbUsersStories: 0,
+      bugsCount: 0,
+      allBugsCount: 0,
+      timeToFix: '',
+      timeToDetect: '',
+      cascade: [{}],
+      sprint: [{}],
+      userStories: [{}],
+      bugHistories: [{}],
+    };
+    await MvpService.createMvp(mvp);
+    dispatch(close());
+    dispatch(fetchAllMvps());
+  }
+
+  const body = (
+    <div className={classes.paper}>
+      <form onSubmit={submit}>
+        <div className={classes.partition}>
+          {imageUrl && (
+            <img className={classes.img} alt="default value" src={imageUrl} />
```

Comment:
for the css of your image, add "object-fit: cover; this will resolve the problem of stretching, in this case, its better to notify the scrum masters that the best photo format to upload is 16:9.
In this case, when the url is not filled yet, the text for now may be something like: "No image URL has been entered. Click on the "Add Picture" button to add one. (Recommanded photo size: 16:9)"